README.md

```diff
@@ -18,3 +35,43 @@ rosdep install -y --rosdistro $ROS_DISTRO --ignore-src --from-paths src
 source /opt/ros/$ROS_DISTRO/setup.bash
 colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release
 ```
+Source the workspace:
+
+```
+cd ~/ws_driver
+source install/setup.bash
+```
+
+Start the driver:
+
+```
+ros2 launch ur_ros2_control_demos ur5_3_system_position_only.launch.py
+```
+
+Start the `joint_state_controller`:
+
+```
+ros2 control load_start_controller joint_state_controller
+```
+
+Start the `forward_command_controller`:
+
+```
+ros2 control load_start_controller forward_command_controller_position
+```
+## TODO Run a test node (TBD)
+Run a test node which will publish joint commands on /forward_command_controller_position/commands (std_msgs::msg::Float64MultiArray)
```

Comment:
Do you have this test node?

I wrote one in the ros2_control_demos. it is currently under PR ros-controls/ros2_control_demos#53

Reply:
Not yet, but will be added. For now I think it is enough to be able to publish position command. So the other parties can start testing.